lib/charms/mongodb/v0/config_server_interface.py

```diff
@@ -286,6 +293,7 @@ def _on_relation_changed(self, event) -> None:
             return
 
         self.charm.status.set_and_share_status(ActiveStatus())
+        self.charm.mongos_intialised = True
```

Comment:
This was an additional change made to the lib.

This will be used to verify if we have already configured + started the mongos process.